### DISCUSSION TITLE
Will there be a windows version?

### DISCUSSION THREAD

**Discussion by xiongster** [sct/overseerr#625] [General]
If so, any ETA? Running overseerr in docker in windows 10 takes too much ram. 85% of ram out of 8gb is always full.

**Comment by sct** [comment]
That’s just docker reserving memory. It shouldn’t be using that much. Why not restricting how much ram docker can reserve in its settings?

**Comment by dRoskar** [comment]
I would like to reiterate the question. Regardless of memory issues. (besides, running docker on an older machine is a lot)
